demo/segmentation/segment.py

```diff
@@ -47,13 +54,15 @@ def random_color():
     pass
 
 # prediction call
-parameters_input = {'segmentation':True}
-parameters_mllib = {'gpu':True,'gpuid':0}
-parameters_output = {}
+parameters_input = {'segmentation':True,'scale':args.scale,'rgb':args.rgb}
+parameters_mllib = {'segmentation':True}
```

Comment:
Do we really need to have `segmentation` twice here?

Reply:
No we don't, will fix.